pycontrails/core/interpolation.py

```diff
@@ -71,7 +71,9 @@ def __init__(
 
         self.grid = points
         self.values = values
-        self.method = method
+        # TODO: consider supporting updated tensor-product spline methods
+        # see https://github.com/scipy/scipy/releases/tag/v1.13.0
+        self.method = _pick_method(scipy.__version__, method)
```

Comment:
Can you make an issue for this? Thanks for handling for now

Reply:
https://github.com/contrailcirrus/pycontrails/issues/172